app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -1,8 +1,19 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
-import { generateMainImage } from '../actions';
 
 // const ImageManagement = ({ onExit }) => (
+
+const handleInputText = ({ label, id, value, onKeyUp }) => {
```

Comment:
not sure it's the correct name, this is a component after all. The name suggests an action

Reply:
yeah. it  is changed as `TextField`